LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 CUMoellerLab
```

Comment:
We should probably leave this as CUMoellerLab; I don't think Andy himself owns the copyright in this case unless we discuss otherwise.

Reply:
I changed it because I wasn't sure what entity CUMoellerLab actually refers to. A github account? Is that meaningful in any sort of legal sense? Or does it refer to Andy's lab itself?

Reply:
We could also change to "the authors". Copyright is implicit, so in the absence of any formal license statement that would be the case anyway. Assigning MIT license permissions from "the authors" would clarify legally permissible use without also transferring ownership.

Reply:
That sounds reasonable. I'll make that change.